brayns/network/plugin/NetworkManagerEntrypoints.h

```diff
@@ -139,8 +140,11 @@ class NetworkManagerEntrypoints
         plugin.add<ClearLightsEntrypoint>();
         plugin.add<CancelEntrypoint>();
         plugin.add<RegistryEntrypoint>();
-        plugin.add<ExportFramesToDiskEntrypoint>();
-        plugin.add<GetExportFramesProgressEntrypoint>();
+        plugin.add<ExportFramesEntrypoint>();
+
+        auto frameExporter = std::make_shared<FrameExporter>();
```

Comment:
Could be unique ptr with a ref for the progress but it also make sense, no need to change that :).

Reply:
I thought it would be semantically more correct to use a shared_ptr so they can share ownership. 
It would be safe with the unique_ptr as well, since Its impossible get-export-frames-progress can be called after the export-frames-to-disk entrypoint is destroyed, which would result on a segfault. Nonetheless, shared_ptr will completely remove such far fetched case anyway.

Reply:
Yep you are right no need to make dangerous stuff for 1ns optimization